examples/appliance_snmp_v1_trap_destinations.go

```diff
@@ -2,24 +2,39 @@ package main
 
 import (
 	"fmt"
+
 	"github.com/HewlettPackard/oneview-golang/ov"
-	"os"
-	"strconv"
 )
 
 func main() {
 	var (
 		ClientOV *ov.OVClient
 	)
-	apiversion, _ := strconv.Atoi(os.Getenv("ONEVIEW_APIVERSION"))
+	/*
+		apiversion, _ := strconv.Atoi(os.Getenv("ONEVIEW_APIVERSION"))
+		ovc := ClientOV.NewOVClient(
+			os.Getenv("ONEVIEW_OV_USER"),
+			os.Getenv("ONEVIEW_OV_PASSWORD"),
+			os.Getenv("ONEVIEW_OV_DOMAIN"),
+			os.Getenv("ONEVIEW_OV_ENDPOINT"),
+			false,
+			apiversion,
+			"*")
+
+
+	*/
+	config, config_err := ov.LoadConfigFile("config.json")
+	if config_err != nil {
+		fmt.Println(config_err)
+	}
 	ovc := ClientOV.NewOVClient(
-		os.Getenv("ONEVIEW_OV_USER"),
-		os.Getenv("ONEVIEW_OV_PASSWORD"),
-		os.Getenv("ONEVIEW_OV_DOMAIN"),
-		os.Getenv("ONEVIEW_OV_ENDPOINT"),
-		false,
-		apiversion,
-		"*")
+		config.OVCred.UserName,
+		config.OVCred.Password,
+		config.OVCred.Domain,
+		config.OVCred.Endpoint,
+		config.OVCred.SslVerify,
+		config.OVCred.ApiVersion,
+		config.OVCred.IfMatch)
 
 	snmpUser := ov.SNMPv1Trap{
 		Destination:     "192.0.1.14",
```

Comment:
snmp trap destination ip should come from config?

Reply:
Yeah that is random ip